scripts/actions/mobskills/saline_coat.lua

```diff
@@ -16,7 +16,9 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    skill:setMsg(xi.mobskills.mobBuffMove(mob, xi.effect.MAGIC_DEF_BOOST, 50, 0, 60))
+    local power = 100 + (skill:getTP() - 1000) / 100
+
+    skill:setMsg(xi.mobskills.mobBuffMove(mob, xi.effect.MAGIC_DEF_BOOST, power, 0, 60))
```

Comment:
This decays, and it looks like it's hooked up but with the wrong power but that's ok. make a note about it
(Decay is -1/tick and not -5/tick like jimmy's sheet)